src/libraries/Microsoft.Win32.SystemEvents/src/Microsoft.Win32.SystemEvents.csproj

```diff
@@ -8,6 +8,7 @@
   </PropertyGroup>
   <!-- DesignTimeBuild requires all the TargetFramework Derived Properties to not be present in the first property group. -->
   <PropertyGroup>
+    <DefineConstants Condition="$(TargetFramework.StartsWith('netstandard2.0')) or $(TargetFramework.StartsWith('netcoreapp2.0')) or $(TargetFramework.StartsWith('netcoreapp3.0'))">$(DefineConstants);FEATURE_CER</DefineConstants>
```

Comment:
I'd actually turn this condition on its head. If the target framework starts with anything _other_ than `"netcore"`, define the constant. These APIs are no-ops across all versions of .NET Core, so excluding netcoreapp2.0/3.0 from containing the calls is ok.

It also helps make this a bit more robust, as we don't have to worry about "wait, does this project also target NetFramework?" This project does, and Full Framework should get CER calls.

(This comment applies to other .csproj files in this PR as well.)

Reply:
I considered that but chose to go with an approach that uses a fixed list of known, past targets so that future versions are excluded by default.